virtio-queue/src/queue/verification.rs

```diff
@@ -221,6 +453,7 @@ impl kani::Arbitrary for ProofContext {
     }
 }
 
+/// Kani proof harness for VirtIO 1.2 Section 2.7.7.2: Device-to-driver notification suppression.
```

Comment:
Since this is for a different proof we could just add this comment in a new PR.

Reply:
done